src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,107 @@
+import { useContext } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from './Todos.Context';
+import { Status } from '../types/Todo';
+// import { deleteTodo } from '../api/todos';
+
+export const TodoFilter: React.FC = () => {
+  const {
+    todos,
+    setStatus,
+    filteredTodos,
+    // setTodos,
+    status,
+    // setError,
```

Comment:
Remove comments everywhere
```suggestion
    status,
```